src/intervals/conversion.jl

```diff
@@ -17,7 +17,7 @@ convert(::Type{Interval{T}}, x::Bool) where {T} = convert(Interval{T}, Int(x))
 convert(::Type{Interval{T}}, x::Real) where {T} = atomic(Interval{T}, x)
 convert(::Type{Interval{T}}, x::T) where {T<:Real} = Interval{T}(x)
 convert(::Type{Interval{T}}, x::Interval{T}) where {T} = x
```

Comment:
```suggestion
convert(::Type{Interval{T}}, x::Interval{T}) where {T<:Real} = x
```

Reply:
While I see your point, this would make the code harder to maintain if we change the parameter type constrain.

Reply:
You can add a constant and use that instead of `Real`.

```julia
const IT = Real  # type constraint of intervals
```